jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -240,6 +248,39 @@ node {
         isGARelease = hasNewGARelease(componentRelease, latestRelease, productName)
         println "[INFO] latestRelease:${latestRelease}, componentRelease:${componentRelease}, isGARelease:${isGARelease}"
         currentBuild.description = "latest: ${latestRelease}\ncurrent: ${componentRelease}"
+
+        if (releaseTagVar == "") {
+          productVersion = latestRelease
+        }
+
+        println "[INFO] product version: ${productVersion}"
+    }
+
+    stage('Fetch Component Product Version') {
+        when(productVersion == "") {
+          println "[INFO] Retrieving component product version"
+          def templateUrl = "https://raw.githubusercontent.com/${projectOrg}/${projectRepo}/${latestRelease}/${productVersionLocation}"
+
+          if (projectOrg == "integr8ly") {
+            productVersion = sh(returnStdout: true, script: "curl -i '${templateUrl}' | grep '${productVersionIdentifier}' | head -n 1 | awk '{print \$3}'")
+            productVersion = productVersion.replaceAll("[\"\']", "")
+          }
```

Comment:
Is this for other integeratly components other than webapp?

Reply:
yes, those should be for the other integreatly components. This could be used for the webapp in the future but at the moment, the value for the product version within that file in our current release is wrong.